federatedscope/core/monitors/metric_calculator.py

```diff
@@ -162,18 +167,22 @@ def eval_roc_auc(y_true, y_prob, **kwargs):
     return sum(rocauc_list) / len(rocauc_list)
 
 
-def eval_rmse(y_true, y_pred, **kwargs):
```

Comment:
we'd better give up this changes. probability is quite confusing in the context of regression that involves mean squired loss.